libsupport/include/katana/ProgressTracer.h

```diff
@@ -154,6 +165,8 @@ class KATANA_EXPORT ProgressTracer {
   std::shared_ptr<ProgressSpan> default_active_span_ = nullptr;
 };
 
+KATANA_EXPORT ProgressTracer& getTracer();
```

Comment:
Added the global function getTracer(), is that good Roshan?

Reply:
Thanks, that's good. Please use `GetTracer()`. `getSystemNetworkInterface` is outdated in naming style.